composer.json

```diff
@@ -19,17 +19,23 @@
         "slack": "https://zendframework-slack.herokuapp.com",
         "forum": "https://discourse.zendframework.com/c/questions/expressive"
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/webimpress/zend-expressive-router"
```

Comment:
I guess this is here for testing only as it should never be merged.

Reply:
This PR depends on  zendframework/zend-expressive-router#53 and there is no other way (or I don't know it) to test it than including the repository from my branch with changes. Once  zendframework/zend-expressive-router#53 is merged I'll update this (and also other PRs).